packs/data/journals.db/archetypes.json

```diff
@@ -2553,6 +2561,423 @@
mpendium[pf2e.feats-srd.Lie To Me]{Lie To Me} skill feat.</p><hr /><p><strong>Special</strong> You can't select another dedication feat until you've gained two other feats from the Alkenstar Agent archetype.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.Jk6gZzXEABiX5A0S]{Loose Cannon} Feat 4</h2><p><strong>Prerequisites </strong>Alkenstar Agent Dedication</p><hr /><p style=\"box-sizing:border-box;user-select:text;margin:0.5em 0px;min-height:unset;font-weight:500;color:rgb(25, 24, 19);font-family:Roboto, sans-serif;font-size:14px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial\"><span style=\"font-family:Roboto, sans-serif\"><strong>Your unpredictable gunfire often catches enemies off guard or hits unprotected areas. Make a ranged Strike with a firearm against a creature that you haven't already attacked on this turn. You gain a circumstance bonus to damage on this Strike equal to the number of weapon damage dice. The Strike gains the following failure effect.</strong></span></p><hr /><p><strong>Failure</strong> The firearm misfires, but it doesn't cause the other critical failure effects listed below.</p><p><strong>Critical Failure</strong> The firearm misfires and also explodes. It becomes broken, and it deals its normal weapon damage to all creatures in a @Template[type:burst|distance:20] centered on the firearm, with a @Check[type:reflex|dc:resolve(@actor.attributes.classDC.value)|basic:true] against your class DC. This damage includes any from the weapon's fundamental and property runes.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.JEzjyNbpsh05iymG]{Prepare Papers} Feat 7</h2><p><strong>Prerequisites </strong>Alkenstar Agent Dedication, master in Society</p><hr /><p>You're practiced at creating forgeries that establish your outlaw credentials or augment your legal authority. You can @Compendium[pf2e.actionspf2e.Create Forgery]{Create Forgery} as an exploration activity. It takes you about 1 hour per page of the document that you're forging. This time is reduced to 10 minutes per page if you're legendary in Society.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.qbFRPTHP96Q9kGpk]{Flash Your Badge} Level 10</h2><p><strong>Prerequisites </strong>Alkenstar Agent Dedication, master in Intimidation</p><hr /><p>You make a fearsome show of your authority. Roll Intimidation checks to @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} each creature in a @Template[type:cone|distance:30]. When you do so, Demoralize gains the visual trait, and creatures are affected if they can see you. If a target creature hasn't broken a law in the past week (as determined by the GM), the result of your check against that creature is one degree of success worse than the result you rolled.</p>",
```

Comment:
All of these have some changes that need made. `<span id=\"ctl00_RadDrawer1_Content_MainContent_DetailedOutput\">` and the connected `</span>` tag need to go.

The style tags and other stuff like

```
<p style=\"box-sizing:border-box;user-select:text;margin:0.5em 0px;min-height:unset;font-weight:500;color:rgb(25, 24, 19);font-family:Roboto, sans-serif;font-size:14px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial\"><span style=\"font-family:Roboto, sans-serif\">
```

Also need to go. Just remove the style from the paragraph tags and delete the span tags entirely. The only span tags we use are to float content to the right and the action icons.